### ISSUE TITLE
Benchmark: ESS and Uniffle

### ISSUE THREAD

**Issue by zuston** [apache/incubator-uniffle#81]
Could the community provide the official benchmark with ESS and uniffle?

**Comment by colinmjj** [comment]
@zuston You can refer this [blog](https://cloud.tencent.com/developer/article/1943179) for the benchmark related.

**Comment by zuston** [comment]
Attach the google doc about test result in our internal cluster： https://docs.google.com/document/d/1nmHMBEaa4lHfgQkdlYokTtXt12F5vRJiQ2MCmTQbW6k/edit#heading=h.b1udpb9l28w7

I can't achieve the same result from the blog. Could u help check this? @colinmjj @jerqi

**Comment by colinmjj** [comment]
@zuston The benchmark of blog is based on Spark 2.4.6.
If there has no random disk IO problem with ESS, Uniffle is expected has **poor performance** than ESS

**Comment by zuston** [comment]
Yes, you are right. @colinmjj . I tested in 5TB terasort, ESS directly failed when shuffle read due to 100% diskio. But uniffle looks good.

I have updated latest result in google doc. @colinmjj @jerqi . I think maybe it would be better to attach the result to github issue.
